### ISSUE TITLE
RESTeasy conflict

### ISSUE THREAD

**Issue by electrobabe** [matomo-org/piwik-java-tracker#23]
when I add the maven dependency 
```
    <dependency>
        <groupId>org.piwik.java.tracking</groupId>
        <artifactId>piwik-java-tracker</artifactId>
        <version>1.2</version>
    </dependency>
```
to my Spring / RESTeasy app, I always get the error
```ERROR [RMI TCP Connection(8)-127.0.0.1] o.s.w.c.ContextLoader - [] - Context initialization failed
java.lang.RuntimeException: RESTEASY013050: RESTeasy Provider Factory is null, do you have the ResteasyBootstrap listener configured?
	at org.jboss.resteasy.plugins.spring.SpringContextLoaderSupport.customizeContext```

any ideas how to fix that?

**Comment by electrobabe** [comment]
ok, it works when I just remove all that J2EE dependencies from your maven dependency:

**Comment by mattab** [comment]
Thanks @electrobabe for the suggestion. Maybe you'd be able to create a pull request? Thanks!

**Comment by dve** [comment]
Is there a chance to get this merged?

**Comment by tholu** [comment]
@dve @electrobabe Should be resolved with #29. Pending an upcoming release. Closing this issue now.
